application/src/main/java/org/togetherjava/tjbot/commands/filesharing/GistRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.togetherjava.tjbot.commands.filesharing;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.jetbrains.annotations.NotNull;
+
+final class GistRequest {
```

Comment:
make this class a record

Reply:
what about GistResponse?

Reply:
fields of a record r final, and `GistResponse` class have a set method so u won't be able to change the value after creating an instance if using record

Reply:
yea true